### DISCUSSION TITLE
[Feature] exclude albums from being shown in asset list/grid

### DISCUSSION THREAD

**Discussion by twoez** [immich-app/immich#3645] [Feature Request]
### The feature

Is it possible to somehow hide certain albums from being shown in the main photo list? In my WhatsApp I get lots of photos and videos. This is automatically added to the asset grid.

I installed latest Android APK from Google Play store.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
You can double tap on an album in the album selection view for upload to exclude an album

**Comment by twoez** [comment]
I know I can exclude albums from being uploaded to the server, but what I meant is how to exclude albums on device itself from to being shown in the photo list of the app locally. WhatsApp is detected automatically.

**Comment by bo0tzz** [comment]
Duplicate of #1236
